dotnet/src/SemanticKernel.Skills/Skills.FileCompression/Skills.FileCompression.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))</RepoRoot>
+  </PropertyGroup>
+
+  <Import Project="$(RepoRoot)/dotnet/nuget/nuget-package.props" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.SemanticKernel.Skills.FileCompression</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Skills.FileCompression</RootNamespace>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- NuGet Package Settings -->
+    <PackageId>Microsoft.SemanticKernel.Skills.FileCompression</PackageId>
```

Comment:
I'd rather name this something more general, like "Skills.FileIO" so that it's not such a single-purpose nuget package. We could add many more file-related skills to this package. (Same for namespace across the project)

Reply:
Would it make sense to rename Skills.Document to Skills.FileIO and then add this functionality in there?

Reply:
Probably not -- Document is about OpenXML (Office Documents). It's already a published NuGet package -- best not to remove it arbitrarily.

Reply:
Moved to samples/skills to avoid creating a Nuget for now - Close this conversation if this works for now

Reply:
Moved to samples/dotnet as per @shawncal 's request